dask/array/linalg.py

```diff
@@ -33,7 +33,7 @@ def _nanmin(m, n):
     return k_1 if np.isnan(k_0) else k_0
 
 
-def tsqr(data, name=None, compute_svd=False, _max_vchunk_size=None):
+def tsqr(data, name=None, compute_svd=False, use_recursion=True, _max_vchunk_size=None):
```

Comment:
The `_max_vchunk_size` parameter is both private and undocumented.  Should it be removed?

Reply:
When would a user want to modify the `use_recursion` flag?  Putting on my naive-user hat it's not clear to me when I should use this or how.  Do we need it?  Are there good reasons to avoid it sometimes?

Reply:
Not sure... If a user wanted to not use recursion, it would mean a possibly big single core compute. They would be going in with eyes open I guess. I've no objection to removing it though.

Reply:
If there isn't a pretty good reason to have the option I'm inclined to remove it.  From a project maintenance perspective it's much easier to add options back in than remove them from the API.